src/app/components/header/header.controller.js

```diff
@@ -1,18 +1,38 @@
 'use strict';
 
-function HeaderController($log, $rootScope) {
-    'ngInject';
+export default class HeaderController{
+    constructor ($rootScope) {
+        'ngInject';   
+        this.menu = [
+            { name: 'Sign in', link: 'sign-in' }, 
+            { name: 'Sign up', link: 'sign-up' }
+        ];
+            
+        if ($rootScope.currentUser) {   
+            this.currentUser = $rootScope.currentUser;
+        }
 
-    if ($rootScope.currentUser) {
-        this.currentUser = $rootScope.currentUser;
-    }
-
-    $rootScope.$watch('currentUser', currentUser => {
-        this.currentUser = currentUser;
-    });
+        $rootScope.$watch('currentUser', currentUser => {
+            this.currentUser = currentUser;
 
-    $log.debug('Hello from Header controller!');
-}
-
-export default HeaderController;
+            if (this.currentUser) {
+                this.menuShow();
+            }
+        });
+    }
 
+    menuShow() {
+        if (this.currentUser.role === 'user') {
```

Comment:
Next time change 'user' and 'admin' to roles.USER and roles.ADMIN.
But only after you get these constants from another PR.